bugbug/model.py

```diff
@@ -268,22 +294,35 @@ def train(self, importance_cutoff=0.15):
         # Evaluate results on the test set.
         y_pred = self.clf.predict(X_test)
 
+        if is_multilabel:
+            assert isinstance(
+                y_pred[0], np.ndarray
+            ), "Looks like the predictions aren't multilabel"
```

Comment:
Nit: the assertion message should be assertive :P 
Something like "The predictions should be multilabel"